src/index.js

```diff
@@ -0,0 +1,63 @@
+const state = {
+    tempCount : 0
+};
+
+const changeTempColor = () => {
+    if (state.tempCount <= 49) {
+        document.getElementById("tempCounter").style.color = "teal";
+    } else if (state.tempCount <= 59) {
+        document.getElementById("tempCounter").style.color = "green";
+    } else if (state.tempCount <= 69) {
+        document.getElementById("tempCounter").style.color = "yellow";
+    } else if (state.tempCount <= 79) {
+        document.getElementById("tempCounter").style.color = "orange";
+    } else if (state.tempCount >= 80) {
+        document.getElementById("tempCounter").style.color = "red";
+    } 
+}
```

Comment:
Consider the above logic and how similar the branches are. We look for whether the temperature is within a range, and pick a color accordingly. What if we had a list of records that we could iterate through to find the values. We could set up something like
```js
  const tempRecords = [
    { upperBound: 49, color: "teal"},
    { upperBound: 59, color: "other color"},
    ...
  ];
```

Then we can simply iterate through the list and find the first record that has an upper bound _higher_  than our temperature, then use it as the source of picking the color!